src/test/resources/simple_trec_run_fusion_1.txt

```diff
@@ -0,0 +1,3 @@
+1 Q0 pyeb86on 1 24 reciprocal_rank_fusion_k=60
```

Comment:
put all these files under a `fusion/` directory please.

Reply:
These test files were actually from Daniel early on; I didn't include my test files on the commit. Of course, I could move these to a `fusion/` directory or just delete them.

Reply:
If these files are actually used, then please move. If they're not, then remove.